libvmaf/src/feature/cambi.c

```diff
@@ -242,6 +256,22 @@ static FORCE_INLINE inline void adjust_window_size(uint16_t *window_size, unsign
     (*window_size) = ((*window_size) * input_width) / CAMBI_4K_WIDTH;
 }
 
+void  set_contrast_arrays(const uint16_t num_diffs, uint16_t **diffs_to_consider,
+                          int **diffs_weights, int **all_diffs)
+{
+    *diffs_to_consider = aligned_malloc(ALIGN_CEIL(sizeof(uint16_t)) * num_diffs, 16);
+    *diffs_weights = aligned_malloc(ALIGN_CEIL(sizeof(int)) * num_diffs, 32);
+    *all_diffs = aligned_malloc(ALIGN_CEIL(sizeof(int)) * (2*num_diffs+1), 32);
```

Comment:
These are unchecked mallocs.

Reply:
In the code, some are checked and some unchecked.  Is the preference to use the pattern below for all the aligned mallocs in cambi?

https://github.com/Netflix/vmaf/blob/d70c679c8278d6da054e86975f971e11b8f5050d/libvmaf/src/feature/vif.c#L323-L328

Reply:
That code is from the older float vif. It should be updated, but it is basically deprecated at this point. Better to follow this pattern: https://github.com/Netflix/vmaf/blob/master/libvmaf/src/feature/integer_vif.c#L585-L586

Easiest thing to do is do all your allocs upfront in `.init()` and fail right away if there are problems.

Reply:
done